### PR TITLE
add simple method for getting a no results message

### PR DESCRIPTION
**Pull request by nburka**

### CODE REVIEW

**Comment by pparke** [comment]
Just a comment, this works but if you are just retrieving a prop you normally wouldn't create a new method unless you had to manipulate it somehow.  For passing props down to an inner component you would just add it to the destructure statement at the top of `render()` and then pass it.

**Comment by nburka** [comment]
> Just a comment, this works but if you are just retrieving a prop you normally wouldn't create a new method unless you had to manipulate it somehow. For passing props down to an inner component you would just add it to the destructure statement at the top of render() and then pass it.

We've had a long talk about this and decided to stick with the named method https://silverorange.slack.com/archives/C041VKQTV/p1531493402000243

File: src/AddressAutoComplete.js [hunk 1]
```diff
@@ -61,7 +67,8 @@ export default class AddressAutoComplete extends Component {
           source={this.getSuggestions}
           minLength={4}
           required
-          displayMenu="overlay"
+          displayMenu='overlay'
```
Comment (gauthierm) [inline src/AddressAutoComplete.js:70]: prettier should use double quotes here

**Comment by gauthierm** [comment]
https://github.com/silverorange/more-accessible-auto-complete/pull/2

**Pull request merged**